lua/drop/config.lua

```diff
@@ -1,5 +1,55 @@
 local M = {}
 
+-- Calculate the date of Easter on a given year via the computus method.
+-- https://en.wikipedia.org/wiki/Date_of_Easter
+function M.calculate_easter(year)
+  -- Step 1: Calculate the Golden Number of the year, which is used to determine the date of the full moon.
+  local goldenNumber = year % 19
+
+  -- Step 2: Calculate century, year of the century, and number of years that have leap years.
+  local century = math.floor(year / 100)
+  local yearOfCentury = year % 100
+  local leapYears = math.floor(century / 4)
+
+  -- Step 3: Calculate skipped leap years.
+  local skippedLeapYears = century % 4
+
+  -- Step 4: Determine the leap year corrections.
+  local leapYearCorrections = math.floor((century + 8) / 25)
+  local additionalLeapYearCorrection = math.floor((century - leapYearCorrections + 1) / 3)
+
+  -- Step 5: Calculate the "epact" which represents the age of the moon on January 1.
+  local epact = (19 * goldenNumber + century - leapYears - additionalLeapYearCorrection + 15) % 30
+
+  -- Step 6: Calculate the number of days from March 1 to the Paschal full moon.
+  local daysToFullMoon = math.floor(yearOfCentury / 4)
+  local correctionFactor = yearOfCentury % 4
+  local daysToSunday = (32 + 2 * skippedLeapYears + 2 * daysToFullMoon - epact - correctionFactor) % 7
+
+  -- Step 7: Calculate the lunar month correction.
+  local lunarMonthCorrection = math.floor((goldenNumber + 11 * epact + 22 * daysToSunday) / 451)
+
+  -- Step 8: Determine the month and day of Easter.
+  local month = math.floor((epact + daysToSunday - 7 * lunarMonthCorrection + 114) / 31)
+  local day = ((epact + daysToSunday - 7 * lunarMonthCorrection + 114) % 31) + 1
+
+  return { month = month, day = day }
+end
+
+-- Calculate the date of US Thanksgiving on a given year.
+function M.calculate_us_thanksgiving(year)
+  local thanksgiving = os.time({ year = year, month = 11, day = 1 })
+  local weekday = os.date("*t", thanksgiving).wday
+  local offset = (11 - weekday) % 7
+  local day = 1 + offset + (3 * 7) -- Fourth Thursday of November.
+  return { month = 11, day = day }
+end
```

Comment:
Exposed both of these functions so that the user could invoke them from their own config if needed.